app/services/task-service.ts

```diff
@@ -89,6 +95,25 @@ export class TaskService extends ServiceBase {
         return getOnceProxy(this.get(jobId, taskId, options));
     }
 
+    public getMultiple(jobId: string, taskIds: string[], properties?: string): Observable<Task[]> {
```

Comment:
i don't want it to update the cache, or it will slam the existing task entities in there.